announcements/global_council_election_2020/social_media_announcement.md

```diff
@@ -0,0 +1,9 @@
+# Copy for social media announcement for registering to be a PyLadies member
+
+
+- **Headline:** We're expanding  !
+- **Body :**  PyLadies member registration is now OPEN !  If you're interested in getting involved and making a difference in the community, then we’d love to hear from you !
+- **Footer :** Click the link below for more info (_insert link_)
```

Comment:
Perhaps the link is to the blog post? It has links to all the forms
https://www.pyladies.com/blog/Announcement-for-Registering-PyLadies-Members/announcement-for-registering-pyladies-members/

Reply:
@Mariatta ok sure